### PR TITLE
AGDIGGER-57 - fetch artifacts support

### PR DESCRIPTION
**Pull request by wtrocki**
# Motivation 

Allow to fetch build artifacts artifacts

Review @aliok @luigizuccarelli @matzew

### CODE REVIEW

**Comment by matzew** [comment]
:+1:

**Comment by aliok** [comment]
should we have some unit tests?
we have some stuff on Java client.
@wtrocki @matzew WDYT?

File: README.md [hunk 1]
```diff
@@ -26,14 +26,17 @@ digkins job build my-androip-build
...
 
-  Setup jenkins credentials and login into jenkins                          
+  Setup jenkins credentials and login into jenkins 
   digkins login <url> [user] [password] 
 
   Stream jenkins logs for triggered build
   digkins log <job> <queueid>       
+  
+  Get job artifacts for specified build
+  artifact <job> <buildNumber>
```
Comment (aliok) [inline README.md]: `digkins artifact <job> bla`

**Comment by wtrocki** [comment]
@aliok 
We can write some basic unit tests to cover all commands - not just this one. 
Let's sync on that to figure out what testing framework we should use. I do not have any preference - we can use Mocha etc.

**Comment by aliok** [comment]
@wtrocki I personally like these:
- Mocha
- Chai (+Chai strings, Chai promise etc.)
- Mockery OR Sinon

**Comment by aliok** [comment]
I think we can merge this one.
Maybe write unit tests in a separate PR? It is up to you.
You introduce the tests.

**Comment by wtrocki** [comment]
@aliok Have you verified this? Is solution ok for you. Going to create new ticket to cover basic test base.

**Comment by aliok** [comment]
@wtrocki Yes I have verified it locally. It is perfect: how it should be!

**Pull request merged**